spec.html

```diff
@@ -5740,7 +5740,7 @@ <h1>
       </h1>
       <dl class="header">
         <dt>description</dt>
-        <dd>It returns _argument_ converted to a Number value if it is a String representation of a Number that would be produced by ToString, or the string *"-0"*. Otherwise, it returns *undefined*.</dd>
+        <dd>If _argument_ is either *"-0"* or exactly matches the result of ToString(_n_) for corresponding Number value _n_, it returns the respective Number value. Otherwise, it returns *undefined*.</dd>
```

Comment:
```suggestion
        <dd>If _argument_ is either *"-0"* or exactly matches the result of ToString(_n_) for some Number value _n_, it returns the respective Number value. Otherwise, it returns *undefined*.</dd>
```